dist/android/colors.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`dist/` is in a weird spot right now where it is not gitignored (to allow npm to publish this dir), but we should also avoid committing it. 

I've filed a follow up ticket to make this easier to manage in the future: https://meetup.atlassian.net/browse/CONST-16